tests/models/test_instrument.py

```diff
@@ -130,7 +134,9 @@ def test_instrument_offsets_and_filter_array_index():
 
     # for the DECam instrument, the offsets are different
     inst = DECam()
-    inst.name = 'TestInstrument' + uuid.uuid4().hex
+    # Why was this here?  This broke the instrument cache
+    # (get_instrument_instance would return the wrong class)
+    # inst.name = 'TestInstrument' + uuid.uuid4().hex
```

Comment:
I think this was a hack to test if we can generate new sections, instead of loading the ones from the DB based on instrument name. 
I don't think the local instance having the wrong name would mess with the global `get_instrument_instance`.

Reply:
It did get saved to the database somewhere, though.